Documentation/gh-pages/source/index.rst

```diff
@@ -106,10 +121,13 @@ PhD theses. Many of these can be found under the list of KATRIN publications_.
 The `kassiopeia/full` image comes with a JupyterLab installation, can run on Kubernetes based JupyterHubs and is also used for the "try it out online" link above.
 
 
-|More information|_
+`More information`_
+
 
-.. |More information| replace:: **More information**
-.. _More information: https://github.com/KATRIN-Experiment/Kassiopeia/blob/main/Docker/README.md
+.. _`Try it out online`: https://mybinder.org/v2/gh/KATRIN-Experiment/KassiopeiaBinder/HEAD
+.. _`the KATRIN collaboration`: https://katrin.kit.edu
+.. _`our paper`: https://iopscience.iop.org/article/10.1088/1367-2630/aa6950
+.. _`More information`: https://github.com/KATRIN-Experiment/Kassiopeia/blob/main/Docker/README.md
```

Comment:
You delete this file with this Pull Request and move the content to the documentation. So this should be updated as well.  Same goes for the "more information" to Docker images link in the main README.